src/lib/viewers/BaseViewer.js

```diff
@@ -906,7 +906,7 @@ class BaseViewer extends EventEmitter {
 
     disableAnnotationControls() {
         if (this.annotator && this.annotationControls && this.areNewAnnotationsEnabled()) {
-            this.annotator.toggleAnnotationMode(AnnotationMode.NONE);
+            this.annotator.toggleAnnotationMode(AnnotationType.NONE);
```

Comment:
Seems like this change kind of breaks down once it moves outside of the controls, since Mode is more appropriate for the Annotations SDK.